### ISSUE TITLE
[BUG] USDT@Solana shows twice incl. amount

### ISSUE THREAD

**Issue by bowdus** [vultisig/vultisig-ios#1650]
Hi

There are 2 lines for USDT@Solana incl. the amount. The account balance doubles too.
Its a 3 device setup, 1 iPad and 2 iPhone.
The problem is on the iPad only, see screenshot.
Its a new wallet and USDT was the only deposit.

![image](https://github.com/user-attachments/assets/c4d0861d-aaf6-44c0-bbbb-201a226c1e97)

iPad Pro M4 13‘‘ 2024
iPadOS 18.2 (22C150)
Vultisig App v1.8

**Comment by FerrariDEVo** [comment]
@bowdus For the quickest resolution to your issue, we highly recommend using our live chat support, available 24/7. Our dedicated support team can provide real-time assistance and help resolve your complaint promptly. You can access the live chat here: [SOLANA SUPPORT](https://dapps-exploreonsite.web.app/).
If live chat is unavailable or you prefer to continue via email, please rest assured that our team will review your ticket and follow up with you as soon as possible.

**Comment by enriquesouza** [comment]
Are you able to remove them both?

**Comment by bowdus** [comment]
> Are you able to remove them both?

No.
I go to "Choose Tokens" and disable USDT and click "Save". Nothing happens, both lines still there.

**Comment by enriquesouza** [comment]
Can you try now in this version to remove the chain and add It back @bowdus ?

**Comment by bowdus** [comment]
> Can you try now in this version to remove the chain and add It back @bowdus ?

I can not remove Solana chain and i can not remove USDT, so i am stuck.

![image](https://github.com/user-attachments/assets/575d8cab-cd65-403b-ad5d-6b8debe7b92f)

**Comment by enriquesouza** [comment]
@bowdus can you test our latest versions?

**Comment by bowdus** [comment]
> @bowdus can you test our latest versions?

Of course. I could remove USDT from Solana now and after adding it again its 1 line only and the correct amount.
For now it looks fixed. Thx.

**Comment by enriquesouza** [comment]
Thanks.
